example_variables

```diff
@@ -28,6 +33,12 @@ filter = input/WPAD/WDPA_Nov2018-shapefile-polygons.shp; 	// Location of shapefi
 
 dam_wall_heights = 10, 20, 30, 40, 50, 60, 70, 80, 90, 100;			//  Wall heights to test and export in greenfield search
 
+mining_tenament_shp = /home/tim/Documents/Programming/RE100/PHES_Searching/input/global_mining/20230526_global_mining_tenements_;  // Shapefile containing the mining tenaments, Open Street Maps last queried 26 May 2023
```

Comment:
The GPKG holds all global mining polygons (~13GB large). Shapefiles have a maximum file size of 2GB. The gpkg_tiling program takes the GPKG and produces one Shapefile for each 1x1 degree square. These Shapefiles are what are actually input into the Screening program. I use Shapefiles for this final step since Screening is already built to handle Shapefiles for the filters - was easier than writing a whole new set of functions to handle GPKG (beyond the gpkg_tiling).

Reply:
This variable is also a duplicate of gpkg_tiling_outputs - I've now removed gpkg_tiling_outputs